### PR TITLE
Update Cython version

### PR DESCRIPTION
**Pull request by okuta**
Ctyhon version 0.26.0 was released. We should use latest Cython.

### CODE REVIEW

File: run_cupy_install_test.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     'cudnn': docker.cudnn_choices + ['cudnn-latest-with-dummy'],
     'nccl': docker.nccl_choices,
     'numpy': ['1.9', '1.10', '1.11', '1.12'],
-    'cython': [None, '0.21', '0.24', '0.25'],
```
Comment (delta2323) [inline run_cupy_install_test.py]: Do you know why 0.21 is kept, while 0.22 and 0.23 are skipped? Depending on the reason, we should not remove 0.21.

File: run_install_test.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
     'cudnn': docker.cudnn_choices + ['cudnn-latest-with-dummy'],
     'nccl': docker.nccl_choices,
     'numpy': ['1.9', '1.10', '1.11', '1.12'],
-    'cython': [None, '0.20', '0.21', '0.23', '0.24'],
```
Comment (delta2323) [inline run_install_test.py]: ditto (why 0.20 and 0.21 are kept and 0.22 is not?)

File: run_cupy_install_test.py (hunk 1 above)
Comment (okuta) [inline run_cupy_install_test.py]: I reverted old versions

File: run_install_test.py (hunk 2 above)
Comment (okuta) [inline run_install_test.py]: I reverted old versions

**Comment by okuta** [comment]
@delta2323 I fixed.

**Comment by delta2323** [comment]
jenkins, test this please.

**Comment by delta2323** [comment]
LGTM

**Pull request merged**